include/openthread-types.h

```diff
@@ -346,7 +346,6 @@ enum
     OT_IP6_ADDRESS_ADDED    = 1 << 0,  ///< IPv6 address was added
     OT_IP6_ADDRESS_REMOVED  = 1 << 1,  ///< IPv6 address was removed
 
-    OT_NET_STATE            = 1 << 2,  ///< Device state (offline, detached, attached) changed
```

Comment:
The stack up/down indication is not something that changes unless it is explicitly set, so this indication is no longer necessary.